TowerFox/Extensions/UIExtensions.swift

```diff
@@ -303,7 +303,11 @@ extension UIColor {
             blue: CGFloat(arc4random_uniform(256))/255,
             alpha: 1)
     }
-    static let appMainColor = UIColor(rgba: "#465A50")//177FC3
+    static let appMainColor = UIColor(named: "App-primary-color")! //UIColor(rgba: "#465A50")//177FC3
+    static let appOnMainColor = UIColor(named: "App-onprimary-color")!
+    static let appTblHeadBackColor = UIColor(named: "AddProject-view-background")!
+    static let appCollapsTblBackColor = UIColor(named: "Settings-table-head-background")!
+    static let appCollapsTblForeColor = UIColor(named: "Settings-table-head-foreground")!
```

Comment:
Main color doesn't work properly.
When I restarted the app after downloading a project, it looked like screenshot;
![image](https://user-images.githubusercontent.com/44682669/76313183-45ec3f00-62b3-11ea-95fa-5a0ffbda6235.png)
Look at the navigation, weird!

And now, look at this screenshot;
![image](https://user-images.githubusercontent.com/44682669/76313259-60261d00-62b3-11ea-812e-0565333dccb4.png)
It seems that main color wasn't set up correctly.

Reply:
@piotrgabriel0724 
I can't see any changes for it, would you like to check any of missing files in there?